config/application.rb

```diff
@@ -74,5 +80,8 @@ class Application < Rails::Application
     # We create this in application.rb instead of an initializer so that it's
     # guaranteed to be available for other initializers to use.
     config.conjur_config = Conjur::ConjurConfig.new
+
+    # Initialize metrics and clean existing data
+    Monitoring::Prometheus.setup
```

Comment:
✅  Seems like a good location for this since right above it we get access to the config